uk.co.compucorp.civicrm.hrleaveandabsences/templates/CRM/HRLeaveAndAbsences/Form/ManageEntitlements.tpl

```diff
@@ -16,15 +16,15 @@
         <div class="col-sm-4 col-sm-offset-4">
           <div class="btn-group btn-group-sm" data-toggle="buttons">
             <label class="btn btn-default">
-              <input type="radio" name="override-filter" value="1">
+              <input type="radio" class="override-filter" name="override-filter" value="1">
```

Comment:
Removing `name` doesn't break the behaviour?

Reply:
I made a bad migration. Will check if the name is required (it was present waaaay before), but the class should go on the button group, not at the input level.

Reference: https://github.com/civicrm/civihr/blob/dc8e7508c27ca58ac95e1f2b2b23ed055a7735a7/uk.co.compucorp.civicrm.hrleaveandabsences/templates/CRM/HRLeaveAndAbsences/Form/ManageEntitlements.tpl#L18

Reply:
ok, sorry, override-filter is ok. The name is not necessary (checked PHP files, none made reference to this field). Will remove name attribute.